workspaces/ui/src/components/diff/v2/CaptureManagerPage.js

```diff
@@ -136,6 +136,17 @@ function AllCapturesStore(props) {
 export function CaptureManagerPage(props) {
   const routerPaths = useRouterPaths();
 
+  // set scrolling
```

Comment:
@trulyronak any reason this can't just be a CSS style?

Reply:
Totally could - we'd instead be swapping classes. Wasn't sure where to put the class definitions / what policy was on CSS files vs useStyles, happy to change as needed